modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -2398,9 +2398,27 @@ definition {
 	@ignore = "Test Stub"
 	@priority = "3"
 	test CanViewDefinitionTitleAtInfoHeader {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanViewDefinitionTitleAtInfoHeader";
 
-		// TODO CanViewDefinitionTitleAtInfoHeader pending implementation
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start-End Connector",
+			connectorOut = "start",
+			key_direction = "bottom");
+
+		PortletEntry.publish();
+
+		Alert.viewSuccessMessageText(successMessage = "Workflow published successfully.");
+
+		ProcessBuilderKaleoDesignerReact.gotoInfoWorkflowDefinition();
+
+		ProcessBuilderKaleoDesignerReact.viewInfoHeaderPresentOnSidebar(tab = "Details");
```

Comment:
@sirfabios the test asks to verify that the definition title is present and not the title of each tab. Please, correct. Thx!